src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -100,6 +100,9 @@ class OrganCellCountSummary(JsonObject):
 
     @classmethod
     def for_bucket(cls, bucket):
+        bucket['key'] = Document.translate_field(bucket['key'],
```

Comment:
Try not to patch argument data structures in place. Modifying arguments is a nasty source of bugs. It's more defensive to make a copy. I fix this in my commit.